util/shared.go

```diff
@@ -170,3 +174,29 @@ func CheckSignature(signature string, pubkey string, message string) (bool, erro
 	}
 	return ecdsa.VerifyASN1(key, hash[:], decodedSig), nil
 }
+
+// SaveKeyPair saves a key pair to a file using the PEM format
+func SaveKeyPair(pubkey string, privkey string, pubfile string, privfile string) error {
+	// saveKeyPair decodes the keys because PEM base64s them too, and decoding means that the pubkey in duck format is the same as the data in the PEM file. (which is nice but an arbitrary decision)
+	d, _ := base64.StdEncoding.DecodeString(privkey)
+	b := pem.EncodeToMemory(&pem.Block{
+		Type:  "DUCKCOIN (ECDSA) PRIVATE KEY",
+		Bytes: d,
+	})
+	if err := ioutil.WriteFile(privfile, b, 0600); err != nil {
+		return err
+	}
+
+	d, _ = base64.StdEncoding.DecodeString(pubkey)
+	b = pem.EncodeToMemory(&pem.Block{
+		Type:  "DUCKCOIN (ECDSA) PUBLIC KEY",
+		Bytes: d,
+	})
+	if err := ioutil.WriteFile(pubfile, b, 0644); err != nil {
+		return err
+	}
+
+	gchalk.BrightYellow("Your keys have been saved to " + pubfile + "(pubkey) and " + privfile + " (privkey)")
```

Comment:
We don’t want any util functions to print anything, move these prints to below this function’s usage in snodes.

Reply:
ok!